tests/unit/misc/userscripts/test_qute_lastpass.py

```diff
@@ -0,0 +1,353 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2014-2020 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for misc.userscripts.qute-lastpass."""
+
+import os
+import json
+import subprocess
+from importlib.machinery import SourceFileLoader
+from importlib.util import spec_from_loader, module_from_spec
+from unittest.mock import MagicMock, call
+
+# qute-lastpass violates naming convention and does not have .py extension
+dirname = os.path.dirname(__file__)
+script_path = os.path.join(
+    dirname, "../../../../misc/userscripts/qute-lastpass")
```

Comment:
This could probably be cleaned up a bit by using `pathlib` (we're trying to switch from `os.path` to `pathlib` anyways, see  #176).

Something like (untested and probably wrong):

```python
repo_root = pathlib.Path(__file__).resolve().parents[4]
script_path = repo_root / 'misc' / 'userscripts' / 'qute-lastpass'
```

Reply:
Thanks. That did not work, but below does.

```
repo_root = pathlib.Path(__file__).resolve().parents[4]
script_path = "{}/misc/userscripts/qute-lastpass".format(repo_root)
```

Reply:
They result in the same path:

```
>>> repo_root = pathlib.Path('/home/florian/proj/qutebrowser/git/tests/unit/misc/userscripts/test_qute_lastpass.py').resolve().parents[4]
>>> repo_root
PosixPath('/home/florian/proj/qutebrowser/git')
>>> str(repo_root / 'misc' / 'userscripts' / 'qute-lastpass')
'/home/florian/proj/qutebrowser/git/misc/userscripts/qute-lastpass'
>>> "{}/misc/userscripts/qute-lastpass".format(repo_root)
'/home/florian/proj/qutebrowser/git/misc/userscripts/qute-lastpass'
```

so you might just be missing a `str()` around it when passing it to `SourceFileLoader`.